cardano-crypto-class/cardano-crypto-class.cabal

```diff
@@ -101,6 +100,8 @@ test-suite test-crypto
   default-language:    Haskell2010
 
   ghc-options:         -Weverything
+                       -Wno-orphans
```

Comment:
I had to remove that and add `{-# OPTIONS_GHC -fno-warn-orphans #-}` to 3 files so that it would pass the stack part of the CI build. It seems stack ignores the `ghc-options` fields in the cabal file.

Reply:
I totally agree that `Wno-orphans` should not be listed there. But again, I prefer `-Wno-orphans` over `-fno-warn-orphans`. For the same reason I use `-XScopedTypeVariables` instead of `-fscoped-type-variables`, the `X` (extension) and `W` (warning) make the purpose of the flag clear, while `f` (flag) is just the most general thing.